.tasks/001-migrate-to-kotlinx-date-time.md

```diff
@@ -0,0 +1 @@
+Migrate all `java.time` usages to `kotlinx.datetime`.
```

Comment:
Are the files in this folder needed in this PR?

The reason I am asking is that his one relates to a different PR (which we discussed would be created in some future)

Also, they are 
a) named `001` and `005` (and where are the remaining agents with the license to kill, namely, 002, 003, 004? KIA?)
b) `001` seems to be a one-time operation, so I wouldn't put it to `master`.